.gitignore

```diff
@@ -1,6 +1,9 @@
 *.swp
 *.beam
+*.bk
 .DS_Store
+**/src
```

Comment:
well there are five `src` dirs in this repo, but we'll consider them to be the exception not the rule